app/src/main/java/com/android/sample/ui/authentication/SignInScreen.kt

```diff
@@ -172,16 +171,11 @@ fun SignInScreen(navigationActions: NavigationActions) {
   val activityContext = LocalContext.current
   val token = stringResource(R.string.default_web_client_id)
 
-  val googleIdOption: GetGoogleIdOption =
-      GetGoogleIdOption.Builder()
-          .setFilterByAuthorizedAccounts(true)
-          .setServerClientId(token)
-          .setAutoSelectEnabled(true)
-          .setNonce(NONCE)
-          .build()
+  val getSignInWithGoogleOption: GetSignInWithGoogleOption =
+      GetSignInWithGoogleOption.Builder(token).build()
 
   val request: GetCredentialRequest =
-      GetCredentialRequest.Builder().addCredentialOption(googleIdOption).build()
```

Comment:
Can you explain what was the issue ? What exactly was the problem with the previous `Sign In` and where it came from and how it was solved ?

Reply:
The past class wouldn't allow you to add an account, it would simply quit once it wouldn't find any. This means on machines without google accounts (ex: emulators), it would not doing anything and fail.